### PR TITLE
Add image list of the Argo CD

### LINKED ISSUE #1930: Put all the images of Argo CD into the image list
In order to make all the images of Argo CD be available in the offline environment. We need to put them into the following file:

```
roles/download/defaults/main.yml
```

/assign
/area devops

### PR DESCRIPTION
**Pull request by LinuxSuRen**
fix #1930

/area devops

### CODE REVIEW

File: roles/download/defaults/main.yml [hunk 1]
```diff
@@ -126,6 +126,16 @@ s2ioperator_tag: "v3.2.1"
...
 s2itemplates_tag: "v3.2.0"
 
+#argocd
+argocd_repo: "{{ base_repo | default('quay.io/') }}{{ namespace_override | default('argoproj') }}/argocd"
+argocd_tag: v2.3.3
+argocd_applicationset_repo: "{{ base_repo | default('quay.io/') }}{{ namespace_override | default('argoproj') }}/argocd-applicationset"
+argocd_applicationset_tag: v0.4.1
+argocd_dex_repo: "{{ base_repo | default('ghcr.io/') }}{{ namespace_override | default('dexidp') }}/dex"
+argocd_dex_tag: v2.30.2
+argocd_redis_repo: "{{ base_repo }}{{ namespace_override }}/redis"
```
Comment (pixiake) [inline roles/download/defaults/main.yml:136]: Can argocd use the same redis as kubesphere?

Comment (LinuxSuRen) [reply roles/download/defaults/main.yml:136]: I'm not sure about this. I guess more tests are needed if we let them use the same image version.

**Pull request merged**